docs/src/_reference/settings.md

```diff
@@ -464,14 +464,20 @@ The host to bind to.
 Together with the [`ui.bind_port` setting](#ui-bind-port), this setting corresponds to
 [Gunicorn's `bind` setting](https://docs.gunicorn.org/en/stable/settings.html#bind).
 
+### <a name="ui-update-check"></a>`ui.update_check`
+
+- [Environment variable](/guide/configuration#configuring-settings): `MELTANO_UI_UPDATE_CHECK`
+- Type: `boolean`
+- Default: `true`
+
+Whether to check for newer Meltano versions and display update notifications in the UI.
+
 #### How to use
 
 ```bash
-meltano config meltano set ui bind_host 127.0.0.1
```

Comment:
Are we intending to delete these? https://docs.meltano.com/reference/settings#meltano-ui-server

Reply:
Hi, yeah, I'll see if I can finish it up this week. Having a lot on my plate at the moment!